diff --git a/dist/handleUpload.js b/dist/handleUpload.js
index d40221cd6a1f4d9865cdb0943d9e7e3dcd055333..89f1cf0ca2ebe7ed9cb7621853637ff0a9e28a58 100644
--- a/dist/handleUpload.js
+++ b/dist/handleUpload.js	
@@ -9,7 +9,7 @@ export const getHandleUpload = ({ acl, bucket, getStorageClient, prefix = '' })=
         const fileBufferOrStream = file.tempFilePath ? fs.createReadStream(file.tempFilePath) : file.buffer;
         if (file.buffer.length > 0 && file.buffer.length < multipartThreshold) {
             await getStorageClient().putObject({
-                ACL: acl,
+                ...(acl && { ACL: acl }),
                 Body: fileBufferOrStream,
                 Bucket: bucket,
                 ContentType: file.mimeType,
@@ -20,7 +20,7 @@ export const getHandleUpload = ({ acl, bucket, getStorageClient, prefix = '' })=
         const parallelUploadS3 = new Upload({
             client: getStorageClient(),
             params: {
-                ACL: acl,
+                ...(acl && { ACL: acl }),
                 Body: fileBufferOrStream,
                 Bucket: bucket,
                 ContentType: file.mimeType,
